src/write.jl

```diff
@@ -2,28 +2,30 @@
 ##### utilities
 #####
 
-_edf_write(io::IO, value::Union{String,Char}) = Base.write(io, value)
-_edf_write(io::IO, date::Date) = Base.write(io, uppercase(Dates.format(date, dateformat"dd-u-yyyy")))
-_edf_write(io::IO, date::DateTime) = Base.write(io, Dates.format(date, dateformat"dd\.mm\.yyHH\.MM\.SS"))
-_edf_write(io::IO, x::Integer) = Base.write(io, string(x))
-_edf_write(io::IO, x::AbstractFloat) = Base.write(io, string(isinteger(x) ? trunc(Int, x) : x))
+_edf_repr(value::Union{String,Char}) = value
+_edf_repr(date::Date) = uppercase(Dates.format(date, dateformat"dd-u-yyyy"))
+_edf_repr(date::DateTime) = Dates.format(date, dateformat"dd\.mm\.yyHH\.MM\.SS")
+_edf_repr(x::Integer) = string(x)
+_edf_repr(x::AbstractFloat) = string(isinteger(x) ? trunc(Int, x) : x)
+_edf_repr(::Missing) = 'X'
```

Comment:
```suggestion
_edf_metadata_repr(::Missing) = 'X'
_edf_metadata_repr(x) = _edf_repr(x)
```

This behavior is specific to the `PatientID`/`RecordingID` representation, so it should be handled there (as it was previously). This lets us do that while keeping the more concise version here